### ISSUE TITLE
bug:

### ISSUE THREAD

**Issue by XynoxTheDev** [Dude-Perfect-Discord-Bot/Dude-Perfect#140]
Causing uptime issues!

**Comment by officialpiyush** [comment]
is it heroku? Elaborate please

**Comment by XynoxTheDev** [comment]
> is it heroku? Elaborate please

Not sure, it's just like the previous 403 unauthorised error. This might be because of outdated dependency or intents! Atm I'm working of v3 which should remove this issue, if there is any in the code.

**Comment by XynoxTheDev** [comment]
> is it heroku? Elaborate please

Not sure, it's just like the previous 403 unauthorized error. This might be because of outdated dependency or intents! Atm, I'm working on v3 which should remove this issue, if there is any in the code. https://github.com/Dude-Perfect-Discord-Bot/Dude-Perfect/discussions/141#discussion-3573533

**Comment by officialpiyush** [comment]
I see, are we rewriting the bot in v3 then?

**Comment by XynoxTheDev** [comment]
> I see, are we rewriting the bot in v3 then?

Yes! That's why I have been studying examples of sapphire and started refactoring the code for v3. 
https://github.com/Dude-Perfect-Discord-Bot/Dude-Perfect/tree/Sapphiredev

**Comment by officialpiyush** [comment]
o, makes sense
